### DISCUSSION TITLE
Overview Panel Based on qmd / ipynb Section Titles

### DISCUSSION THREAD

**Discussion by s3alfisc** [posit-dev/great-tables#482] [Q&A]
Hi all, 

I was wondering if it was possible to add an overview / section panel on the left or right hand side of a page which allows to easily navigate sections; similar to the "on this page" feature in r `pkgdown`. See for example the `fixest` docs [here](https://lrberge.github.io/fixest/articles/fixest_walkthrough.html). 

I know that there are ways to do this by dividing a qmd file into chunks (this seems to be what is happening in `quartodoc's` own docs), but this is not an ideal solution when working with code and the input of the next chunk depends on data created in the chunk before. 

Thanks, and all the best, 

Alex

**Comment by s3alfisc** [comment]
Oh wow, it's so easy! I just need to add

```
toc: true
toc-title: "On this page"
toc-location: left
toc-depth: 2
```
and that's it!

![image](https://github.com/user-attachments/assets/568d77c0-ed24-4fcd-82b5-405dd22ef0df)

**Comment by s3alfisc** [comment]
Wow, sorry, this was a question not for great tables but for quartodoc. Urgh.

**Reply by machow** [reply]
No worries -- glad you figured it out! We maintain both Great Tables and quartodoc, so it was still helpful to read through how you solved the problem :p

**Reply by s3alfisc** [reply]
Thanks @machow - we've turned into great-tables power **and** quartodoc power users over at pyfixest and I just confused the two 😅 
Here is the [section of the pyfixest docs ](https://py-econometrics.github.io/pyfixest/table-layout.html) where we use both of them together (curtesy of @dsliwka); and I have sucessfully managed to add a "on this page" sidebar as desired =)
